custom/icds_reports/utils/aggregation_helpers/distributed/agg_gov_dashboard.py

```diff
@@ -0,0 +1,194 @@
+from dateutil.relativedelta import relativedelta
+
+from custom.icds_reports.const import AGG_GOV_DASHBOARD_TABLE
+from custom.icds_reports.utils.aggregation_helpers import transform_day_to_month, month_formatter
+from custom.icds_reports.utils.aggregation_helpers.distributed.base import BaseICDSAggregationDistributedHelper
+
+
+class AggGovDashboardHelper(BaseICDSAggregationDistributedHelper):
+    helper_key = 'agg-gov-dashboard'
+    base_tablename = AGG_GOV_DASHBOARD_TABLE
+
+    def __init__(self, month):
+        self.month_start = transform_day_to_month(month)
+        self.prev_month_start = self.month_start - relativedelta(months=1)
+        self.next_month_start = self.month_start + relativedelta(months=1)
+
+    def aggregate(self, cursor):
+        drop_table_query = self.drop_table_if_exists()
+        cursor.execute(drop_table_query)
+
+        create_table_query, create_params = self.create_child_table()
+        cursor.execute(create_table_query, create_params)
+
+        agg_query, agg_params = self.aggregate_query()
+
+        cursor.execute(agg_query, agg_params)
+
+        update_queries = self.update_queries()
+
+        for query, param in update_queries:
+            cursor.execute(query, param)
+
+        index_query = self.indexes()
+        cursor.execute(index_query)
+
+    def _tablename_func(self, month=None):
```

Comment:
nit: i think this looks cleaner as a property (probably just called `tablename`) especially since the month parameter is unused

Reply:
Make sense